scripts/validate-extension-inputs.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env babel-node
+
+/**
+ * @fileoverview
+ * Script to validate extension block input placeholders
+ */
+
+import fs from 'fs';
+import path from 'path';
+import async from 'async';
+import assert from 'assert';
+import locales from '../src/supported-locales.js';
+
+// Globals
+const JSON_DIR = path.join(process.cwd(), '/editor/extensions');
+const source = JSON.parse(fs.readFileSync(`${JSON_DIR}/en.json`));
+
+// Matches everything inside brackets, and the brackets themselves.
+// e.g. matches '[MOTOR_ID]', '[POWER]' from 'altera a potência de [MOTOR_ID] para [POWER]'
+const blockInputRegex = /\[.+?\]/g;
+
+let numTotalErrors = 0;
+
+const validateExtensionInputs = (translationData, locale) => {
+    let numLocaleErrors = 0;
+    for (const block of Object.keys(translationData)) {
+        const englishBlockInputs = source[block].match(blockInputRegex);
+
+        if (!englishBlockInputs) continue;
+
+        // If null (meaning no matches), that means that English block inputs exist but translated ones don't.
+        // Coerce it to an empty array so that the assertion below fails, instead of getting the less-helpful error
+        // that we can't call Array.includes on null.
+        const translatedBlockInputs = translationData[block].match(blockInputRegex) || [];
+
+        for (const input of englishBlockInputs) {
+            // Currently there are enough errors here that it would be tedious to fix an error, rerun this tool
+            // to find the next error, and repeat. So, catch the assertion error and add to the number of total errors.
+            // This allows all errors to be displayed when the command is run, rather than just the first encountered.
+            try {
```

Comment:
Thank you! We should probably do this for the other validation checks as well. Stopping on the first one isn't bad with the established languages. On the other hand, whenever we add a new language, invariably there are mistakes on 5-10 strings, and that's tedious.